webapp/src/Entity/Submission.php

```diff
@@ -90,11 +90,13 @@ class Submission extends BaseApiEntity implements ExternalRelationshipEntityInte
     private ?string $entry_point = null;
 
     #[ORM\Column(
-        nullable: false,
-        options: ['comment' => 'Whether this submission was imported during shadowing but had an error while doing so.']
+        nullable: true,
+        options: ['comment' => 'If this submission was imported during shadowing but had an error while doing so, the error message.']
```

Comment:
```suggestion
        options: ['comment' => 'The error message for submissions which got an error during shadow importing.']
```

I find the sentence hard to read otherwise.